### PR TITLE
api: addition of restart encoding task 

### PR DESCRIPTION
**Pull request by switowski**

### CODE REVIEW

**Comment by switowski** [comment]
No longer WIP

File: cds_sorenson/api.py [hunk 1]
```diff
@@ -143,26 +119,80 @@ def get_encoding_status(job_id):
...
+    try:
+        stop_encoding(job_id)
+    except SorensonError:
+        # If we failed to stop the encoding job, ignore it - in the worst
+        # case the encoding will finish and we won't use the file.
+        pass
+    return start_encoding(input_file, preset_name)
+
+
+def batch_restart_encoding(jobs_ids, input_files, presets_names):
```
Comment (egabancho) [inline cds_sorenson/api.py]: I see the same issue here as in [`batch_start_encoding`](https://github.com/CERNDocumentServer/cds-sorenson/pull/5#discussion_r85533909)
Also I might add a small pre-condition here to check that the job list and the presets have the same length, WDYT?

**Comment by switowski** [comment]
@drjova @egabancho it's ready to be merged

**Comment by egabancho** [comment]
you might want to take a look at travis first ☺️

**Comment by switowski** [comment]
it's not failing, that's a good sign

**Comment by egabancho** [comment]
https://travis-ci.org/CERNDocumentServer/cds-sorenson

**Comment by egabancho** [comment]
Do you think it might be also a good idea to split the commit you have now into two, one for the refactoring you did and the other one for the actual restart work

**Comment by switowski** [comment]
> Do you think it might be also a good idea to split the commit you have now into two, one for the refactoring you did and the other one for the actual restart work

Done.
LGTM, let's waffle it!

**Comment by drjova** [comment]
waiting travis 🦄

**Comment by egabancho** [comment]
`api: addition of restart encoding task` -> `api: restart encoding task addition` ?

**Comment by switowski** [comment]
`api: restart encoding task addition` this sounds like 4 random words not connected to each other :p

**Comment by switowski** [comment]
ping (I don't think we should change the commit message, but if there are more people in favor of that, I will  feel forced to do this :) )

**Pull request merged**